src/trackers/PSS.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+# import discord
+import asyncio
+import requests
+import platform
+from str2bool import str2bool
+
+from src.trackers.COMMON import COMMON
+from src.console import console
+
+
+class PSS():
+    """
+    Edit for Tracker:
+        Edit BASE.torrent with announce and source
+        Check for duplicates
+        Set type/category IDs
+        Upload
+    """
+
+    ###############################################################
+    ########                    EDIT ME                    ######## noqa E266
+    ###############################################################
+
+    # ALSO EDIT CLASS NAME ABOVE
+
+    def __init__(self, config):
+        self.config = config
+        self.tracker = 'PSS'
+        self.source_flag = 'PSS'
+        self.upload_url = 'https://privatesilverscreen.cc/api/torrents/upload'
+        self.search_url = 'https://privatesilverscreen.cc/api/torrents/filter'
+        self.signature = '\n[center][url=https://privatesilverscreen.cc/pages/1]Please Seed[/url][/center]'
+        self.banned_groups = ['4K4U', 'AROMA', 'd3g', 'edge2020', 'EMBER', 'EVO', 'FGT', 'NeXus', 'ION10', 'iVy', 'Judas', 'LAMA', 'MeGusta', 'nikt0', 'OEPlus', 'OFT', 'OsC', 'PYC',
+                              'QxR', 'Ralphy', 'RARBG', 'RetroPeeps', 'SAMPA', 'Sicario', 'Silence', 'STUTTERSHIT', 'Tigole', 'TSP', 'TSPxL','Will1869', 'x0r', 'YIFY', 'core', 'ZMNT', 
+                              'msd', 'nikt0', 'aXXo', 'BRrip', 'CM8', 'CrEwSaDe', 'DNL', 'FaNGDiNG0', 'FRDS', 'HD2DVD', 'HDTime', 'Leffe', 'mHD', 'mSD', 'nHD', 'nSD', 'NhaNc3', 'PRODJi',
+                              'RDN', 'SANTi', 'ViSION', 'WAF', 'YTS', 'FROZEN', 'UTR', 'Grym', 'GrymLegacy', 'CK4', 'ProRes', 'MezRips', 'GalaxyRG', 'RCDiVX', 'LycanHD']
+        pass
+
+    async def get_cat_id(self, category_name):
+        category_id = {
+            'MOVIE': '1',
+            'TV': '2',
+        }.get(category_name, '0')
+        return category_id
+
+    async def get_type_id(self, type):
+        type_id = {
+            'DISC': '1',
+            'REMUX': '2',
+            'ENCODE': '3',
+            'WEBDL': '4',
+            'WEBRIP': '5',
+            'HDTV': '6',  
+        }.get(type, '0')
+        return type_id
+
+    async def get_res_id(self, resolution):
+        resolution_id = {
+            '4320p': '1',
+            '2160p': '2',
+            '1080p': '3',
+            '1080i': '4',
+            '720p': '5',
+            '576p': '6',
+            '576i': '7',
+            '480p': '8',
+            '480i': '9'
+        }.get(resolution, '10')
```

Comment:
I think this will give a hard error here if some file doesn't clean match the resolution.  I'm not actually sure what happens site side at other sites that have the existing `'8640p': '10',` option, but not the matching resolution group. I'd feel more comfortable if that resolution group was left in, thanks.

Reply:
I understand and didn't notice that it was needed to be up to 10.  I have added it back in.